include/beast/core/streambuf_readstream.hpp

```diff
@@ -93,8 +93,6 @@ class streambuf_readstream
     static_assert(is_Streambuf<Streambuf>::value,
         "Streambuf requirements not met");
 
-    using error_code = boost::system::error_code;
-
     template<class Buffers, class Handler>
     class read_some_op;
 
```

Comment:
I think there's a misunderstanding. What I have done is create a `beast` level identifier `error_code` (which happens to be an alias for `boost::system::error_code`). The nature of the refactoring is that all code now uses this `beast` level namespace identifier instead of having every class nest its own alias for `boost::system::error_code` out of convenience.